engine_group.go

```diff
@@ -74,6 +74,13 @@ func (eg *EngineGroup) Close() error {
 	return nil
 }
 
+// NewSession returned a group session
+func (eg *EngineGroup) NewSession() *Session {
```

Comment:
I think a name like `NewGroupedSession` or `NewSessionGroup`  will be more clearer to users(instead of comments)

Reply:
No.  This `NewSession` will override the `Engine`'s `NewSession`. So people could use `EngineGroup` like `Engine` and don't change anything.

Reply:
Got it